Sprint 14

a-sheryshov opened the pull request:
feat:add-booking

a-sheryshov committed:
Sprint 14
feat:add-booking

sergeymamonov commented on src/main/java/ru/practicum/shareit/booking/BookingController.java:
```diff
@@ -1,12 +1,50 @@
+    BookingService bookingService;
+
+    @Autowired
+    public BookingController(BookingService bookingService) {
```
Не обязательно писать конструктор руками, если можно воспользоваться аннотацией Lombok @AllArgsConstructor. Посмотреть это и в других классах.

a-sheryshov committed:
Sprint 14
fix:AllArgConstructor vice @Autowired
comment: other classes need superclass constructor call
